### PR TITLE
Search dropdown improvements

### PR DESCRIPTION
**Pull request by edaniels**
Screenshot 2024-10-07 at 10 42 14 AM
Screenshot 2024-10-07 at 10 42 17 AM
Screenshot 2024-10-07 at 10 42 21 AM

### CODE REVIEW

File: assets/js/index.js [hunk 1]
```diff
@@ -61,52 +61,69 @@ function btnVisibility(payload) {
...
...LOWJS_QUEUE || [], s = function (e) { r[e] = function () { var t = Array.prototype.slice.call(arguments); r.load(), o.push([e, null, t]) } }, a = function (e) { r[e] = function () { var t, n = Array.prototype.slice.call(arguments); r.load(); var s = new Promise((function (e, r) { t = { resolve: e, reject: r } })); return o.push([e, t, n]), s } }, u = function (e, t) { r[e] = function () { return t } }; s("_setTargetEnv"), s("closeResourceCenter"), s("init"), s("off"), s("on"), s("prepareAudio"), s("registerCustomInput"), s("remount"), s("reset"), s("setBaseZIndex"), s("setCustomInputSelector"), s("setCustomNavigate"), s("setCustomScrollIntoView"), s("setInferenceAttributeFilter"), s("setInferenceAttributeNames"), s("setInferenceClassNameFilter"), s("setResourceCenterLauncherHidden"), s("setScrollPadding"), s("setShadowDomEnabled"), s("setPageTrackingDisabled"), s("setUrlFilter"), s("openResourceCenter"), s("toggleResourceCenter"), a("endAll"), a("endAllFlows"), a("endChecklist"), a("group"), a("identify"), a("identifyAnonymous"), a("start"), a("startFlow"), a("startWalk"), a("track"), a("updateGroup"), a("updateUser"), u("getResourceCenterState", null), u("isIdentified", !1) } }();
```
[the remark this answers is not in the document]
Comment (edaniels) [reply assets/js/index.js]: heh fixed

File: assets/js/index.js [hunk 2]
```diff
@@ -68,42 +68,59 @@ if(main && scrollBtn) {
...
+    // text typed followed by Enter.
+    search.autocomplete.on('autocomplete:cursorchanged', (event, suggestion) => {
+        cursorUsed = true;
+    });
+    search.autocomplete.on('keydown', (e) => {
+        if (opened && !cursorUsed && e.key === 'Enter' && search.input[0].value !== '') {
+            const query = encodeURIComponent(search.input[0].value);
+            window.location = `${window.location.origin}/search?query=${query}`;
+        }
+    });
```
Comment (npentrel) [inline assets/js/index.js:119]: commenting this out will disable the search page right? I'd like to disable that until I can take a look at making the pages show up more text

Comment (npentrel) [reply assets/js/index.js:119]: doesn't matter if it's otherwise reachable

Comment (npentrel) [inline assets/js/index.js:119]:
```suggestion
    // search.autocomplete.on('keydown', (e) => {
    //     if (opened && !cursorUsed && e.key === 'Enter' && search.input[0].value !== '') {
    //         const query = encodeURIComponent(search.input[0].value);
    //         window.location = `${window.location.origin}/search?query=${query}`;
    //     }
    // });
```

Comment (edaniels) [reply assets/js/index.js:119]: yep I will disable

**Pull request merged**